parent/pom.xml

```diff
@@ -385,6 +386,8 @@
         <jgroups-raft-jgroups-version>4.0.15.Final</jgroups-raft-jgroups-version>
         <jgroups-raft-leveldbjni-version>1.8</jgroups-raft-leveldbjni-version>
         <jgroups-raft-mapdb-version>1.0.8</jgroups-raft-mapdb-version>
+        <jira-guava-version>26.0-jre</jira-guava-version>
+        <jira-rest-client-api-version>5.1.2-2bd0a62e</jira-rest-client-api-version>
```

Comment:
Just noticed this: is this a final version or stuff like that? Because if it's something like a dev build, we cannot rely on their maven repository.

Reply:
We already had not really good experience with external maven repo

Reply:
Changed to atlassian jira client 5.1.0, the differences between 5.1.2 and 5.1.0 are minimal and tests passing.